src/_main_/settings.py

```diff
@@ -207,6 +207,13 @@
 #     }
 # }
 
+CACHES = {
```

Comment:
is this used?  if not remove.

Reply:
George has some incoming changes that will use it